### ISSUE TITLE
Extra {0} output message when having preset with circular inherits

### ISSUE THREAD

**Issue by linquize** [microsoft/vscode-cmake-tools#2118]
Extra {0} output message when having preset with circular inherits.

### Expected:

Output panel
```
[preset] circular.inherits.in.config.preset Circular inherits in configure preset my-preset
```

### Apparent Behavior:

Output panel
```
[preset] circular.inherits.in.config.preset Circular inherits in configure preset {0} my-preset
```

### Platform and Versions

- **Operating System**: Windows
- **CMake Version**: 3.12.18081601
- **VSCode Version**: 1.60
- **CMake Tools Extension Version**: 1.8.1
- **Compiler/Toolchain**: Visual C++ 2017

**Comment by linquize** [comment]
https://github.com/microsoft/vscode-cmake-tools/blob/874cc844e678f491f33962ebd9600a98121f8f53/src/preset.ts#L368
https://github.com/microsoft/vscode-cmake-tools/blob/874cc844e678f491f33962ebd9600a98121f8f53/src/preset.ts#L586
https://github.com/microsoft/vscode-cmake-tools/blob/874cc844e678f491f33962ebd9600a98121f8f53/src/preset.ts#L799
https://github.com/microsoft/vscode-cmake-tools/blob/874cc844e678f491f33962ebd9600a98121f8f53/src/preset.ts#L807
https://github.com/microsoft/vscode-cmake-tools/blob/874cc844e678f491f33962ebd9600a98121f8f53/src/preset.ts#L956
https://github.com/microsoft/vscode-cmake-tools/blob/874cc844e678f491f33962ebd9600a98121f8f53/src/preset.ts#L1169

**Comment by bobbrow** [comment]
Good catch.  @xisui-MSFT it looks like `log.error` is being called as if it were the `localize` API.  Can you fix this?
